Android/AmazonConnectInAppCallingAndroidSample/app/src/main/java/com/amazonaws/services/connect/inappcalling/sample/MainActivity.kt

```diff
@@ -36,9 +50,21 @@ class MainActivity : AppCompatActivity() {
             showGeneralErrorAlert(getString(R.string.error_input_are_required))
             return
         }
+
+        val connectInstanceId = configRepo.getConnectInstanceId()
+        val contactFlowId = configRepo.getContactFlowId()
+        val startWebrtcEndpoint = configRepo.getStartWebrtcEndpoint()
+        val createParticipantConnectionEndpoint = configRepo.getCreateParticipantConnectionEndpoint()
+        val sendMessageEndpoint = configRepo.getSendMessageEndpoint()
```

Comment:
Is this addition an feature improvement(if so, add to README)? or part of Android 15 update?

Reply:
It's just a debugging development thing that allows to load config in local property file, for easy setup and configure the config on the fly